src/pubtools/_marketplacesvm/cloud_providers/base.py

```diff
@@ -208,7 +208,7 @@ def _post_delete(self, push_item: T, **kwargs) -> T:
         Returns:
             The delete result data.
```

Comment:
Can you update info in documentation as well?

Reply:
Updated